samples/oauth_rails_vethealth/app/controllers/session_controller.rb

```diff
@@ -0,0 +1,125 @@
+class SessionController < ApplicationController
+  before_action :require_auth, only: :show
+
+  def login
```

Comment:
Does this method build a url and then return that url? If so, the name "login" for this method is confusing. Maybe something like "generate_oath_url" ?